content/cloud/noops-sql-db.md

```diff
@@ -34,24 +35,21 @@ Once you have access to the private beta, please ensure you have [Spin CLI](./cl
 * Your NoOps SQL Database can hold up 1 GB of data
 
 *Service Limitations*
-* The `--sqlite` flag not supported on `spin cloud deploy`
+* The `--sqlite` flag is not supported on `spin cloud deploy`
 * Fermyon **Cloud** only supports databases named "default" at this time
 
-## How To:
+## How To
 
-### Prepare a Database
+Firstly, if you have not already done so, you will need to [create your first Spin application](https://developer.fermyon.com/spin/quickstart#create-your-first-application). Our blog posts [Announcing Fermyon Cloud’s NoOps SQL Database](https://www.fermyon.com/blog/announcing-noops-sql-db) and [NoOps and Serverless Are the Perfect Pair](https://www.fermyon.com/blog/noops-and-serverless-are-the-perfect-pair) each have an example of how to create a NoOps SQLite Storage application. To tell Spin that we want to use SQLite storage, we only need to [grant SQLite permission to a component in the application’s manifest]((https://developer.fermyon.com/spin/sqlite-api-guide#granting-sqlite-database-permissions-to-components)) (the spin.toml file). For example:
```

Comment:
Do we really need to start from "first create your Spin application"?  To me this takes us away from "here's how to get your SQLite app running in Cloud" to "back to the Getting Started guide again."

Reply:
I have created a separate section for this (making it clear that creating a new application is optional) in case the reader already has an application and they just want to get it running SQLite in Cloud.